### ISSUE TITLE
Error with Notebook #1

### ISSUE THREAD

**Issue by ljbennett62** [watson-developer-cloud/answer-retrieval#15]
After attempting to build using the files from the stack exchange dump script, this error appears:
    “---------------------------------------------------------------------------
    AttributeError                            Traceback (most recent call last)
    <ipython-input-6-f5d9d37d72cb> in <module>()
         30 with open(SOLR_DOCUMENTS_PATH) as data_file:
         31     data = json.load(data_file)
    ---> 32 output = pysolr_client.add(data)
         33 
         34 #Running command that index documents

```
/Users/mwassel/miniconda2/lib/python2.7/site-packages/pysolr.pyc in add(self, docs, boost, fieldUpdates, commit, softCommit, commitWithin, waitFlush, waitSearcher,     overwrite, handler)
    863 
    864         for doc in docs:
--> 865             message.append(self._build_doc(doc, boost=boost, fieldUpdates=fieldUpdates))
    866 
    867         # This returns a bytestring. Ugh.

/Users/mwassel/miniconda2/lib/python2.7/site-packages/pysolr.pyc in _build_doc(self, doc, boost, fieldUpdates)
    788         doc_elem = ElementTree.Element('doc')
    789 
--> 790         for key, value in doc.items():
    791             if key == 'boost':
    792                 doc_elem.set('boost', force_unicode(value))

AttributeError: 'unicode' object has no attribute 'items’”
Going back to the standard, provided data
```

When trying to load the experiment for analysis, I get:
     “---------------------------------------------------------------------------
    IOError                                   Traceback (most recent call last)
    <ipython-input-6-e728f71dcddb> in <module>()
        17 # Solr experiment
        18 solr_experiment_path = os.path.join(experiments_directory, 'exp_solr_only.json')
    ---> 19 solr_experiment = au.RetrieveAndRankExperiment(experiment_file_path=solr_experiment_path)
         20 solr_entries = solr_experiment.experiment_entries
         21 

```
/Users/mwassel/Downloads/answer-retrieval-master/notebooks/analysis_utils.py in __init__(self, experiment_file_path)
    141                 experiment_file_path (str): Path to the experiment file
    142         """
--> 143         obj = json.load(open(experiment_file_path, 'rt'))
    144         self.experiment_entries = obj['experiment_entries']
    145         self.base_url = obj['experiment_metadata']['url']

IOError: [Errno 2] No such file or directory: '/Users/mwassel/Downloads/answer-retrieval-master/notebooks/../data/experiments/exp_solr_only.json’”
```

Is that file being created in the previous step?

Can't  run the rest of the notebook because of previous problem.

**Comment by joe4k** [comment]
It seems the extract_stackexchange_dump.py prints out an older version of the solrDocuments.json
The following modifications in this script seem to create the correct format:

``` python
index = 1
for item in answers:
   # obj = {}
   # obj['doc'] = item
   # add = {}
   # add['add'] = obj
   # output_str = json.dumps(add, sort_keys=True).replace('\n', '')
   output_str = json.dumps(item, sort_keys=True).replace('\n', '')
   # print output_str
   # output_str = output_str.replace('{', '', 1)
   # output_str = ''.join(output_str.rsplit('}', 1))
   index = index + 1
   if index <= len(answers) :
    f2.write(output_str +',' + '\n')
   else:
    f2.write(output_str + '\n')

#f2.write(' "commit" : { }\n')
#f2.write('}')
f2.write(']')
```

**Comment by lkrishnamurthy** [comment]
Merged PR #17, tested. Closing this issue.
